src/lib/job.rs

```diff
@@ -30,6 +30,11 @@ pub struct JobDesc {
     pub status: JobStatus,
 }
 
+pub struct JobInfo {
```

Comment:
why not just add `pid` field into `JobDesc` instead of creating a different struct? This way we only need to pass around one structure instead of two. We can leave the `pid` field empty and populate it later once the process has been spawned

Reply:
That's my goal. I decided to start with a simpler structure in oder to figure out what info I really needed.